### ISSUE TITLE
Test for WCAG success criterion 3.2.2: On input

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#190]
http://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html
Changing the setting of any user interface component does not automatically cause a change of context unless the user has been advised of the behavior before using the component

The intent of this Success Criterion is to ensure that entering data or selecting a form control has predictable effects. Changing the setting of any user interface component is changing some state in the control that will persist when the user is no longer interacting with it. So checking a checkbox or entering text into a text field changes its setting, but activating a link or a button does not. Changes in context can confuse users who do not easily perceive the change or are easily distracted by changes. Changes of context are appropriate only when it is clear that such a change will happen in response to the user's action.

Note: This Success Criterion covers changes in context due to changing the setting of a control. Clicking on links or tabs in a tab control is activating the control, not changing the setting of that control.

Note: What is meant by "component" and "user interface component" here is also sometimes called "user interface element".
## Applicable

Page with one or more forms
## Scope selector

Form
## Failures
- [ ] F36: Automatically submitting a form and presenting new content without prior warning when the last field in the form is given a value (#191 )
- [ ] F37: Launching a new window without prior warning when the status of a radio button, check box or select list is changed ( #192 )
- [ ] F76: Providing instruction material about the change of context by change of setting in a user interface element at a location that users may bypass
## Techniques
- [x] G80: Providing a submit button to initiate a change of context (#69) AND H32: Providing submit buttons ( #68 - implemented)
- [ ] G80: Providing a submit button to initiate a change of context (#69) AND H84: Using a button with a select element to perform an action 
- [ ] G13: Describing what will happen before a change to a form control that causes a change of context to occur is made
- [ ] SCR19: Using an onchange event on a select element without causing a change of context
## Relationships
- if H84 or H32 is true, G80 is true
- G13 is difficult to check automatically, so if G80 is not true, it's a cantTell
- F76 is counterpart of G13
- F36 is counterpart of SCR19
